Signal/src/ViewControllers/ConversationView/TypingIndicatorInteraction.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+import Foundation
+
+@objc(OWSTypingIndicatorInteraction)
+public class TypingIndicatorInteraction: TSInteraction {
+    @objc
+    public static let TypingIndicatorId = "TypingIndicator"
```

Comment:
This is the "stub" `TSInteraction".  Note that it uses a fixed unique id.

Reply:
Should we do more to make sure this is never saved? e.g. override saveWithTransaction?

Reply:
Good idea.